src/types/string.ts

```diff
@@ -13,6 +13,7 @@ export interface ZodStringDef extends z.ZodTypeDef {
   };
 }
 
+// eslint-disable-next-line
```

Comment:
@colinhacks I had to disable this line, because there are multiple different eslint complaints about the regex here
I wanted to first confirm whether you approve modifications of this regex based on eslint recommendations

Reply:
If they don't change the behavior then I'm good with it. I assume the linter wouldn't recommend changes that substantially impact the regex.